networks/testnet/markets.json

```diff
@@ -1,6 +1,7 @@
 [
-    "spot://BTC/USDT",
-    "spot://ETH/USDT",
-    "spot://MATIC/USDT",
-    "spot://SHIB/USDT"
+    "spot://USDC/USDT",
+    "spot://LUBE/USDT",
```

Comment:
I suggest to remote Lube and Linda from this list, until we figure out where to pull the prices